tools/multiqc/main.nf

```diff
@@ -0,0 +1,32 @@
+nextflow.preview.dsl=2
+
+process MULTIQC {
+	
+    // tag "FastQC - $sample_id"
+    
+    input:
+	    path (file)
+		val (outdir)
+		val (multiqc_args)
+        // multiqc_args are best passed into the workflow in the following manner:
+        // --multiqc_args="--exlude STAR --title custom_report_title"
+		val (verbose)
+
+	output:
+	    path "*html",       emit: html
+
+	publishDir "${outdir}/multiqc",
+		mode: "copy", overwrite: true
+
+    script:
+
+		if (verbose){
+			println ("[MODULE] MULTIQC ARGS: " + multiqc_args)
+		}
+
+		"""
+		module load multiqc
```

Comment:
@FelixKrueger use the Nextflow built-in support for this to keep it out of the script block for other users: https://www.nextflow.io/docs/latest/process.html#module

```suggestion
```